public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -324,6 +324,10 @@ export class PanelModel {
     return this.queryRunner;
   }
 
+  hasTitle(): boolean {
+    return this.title && this.title.length > 0;
```

Comment:
you don't need the return type annotation if you bang bang the title : ``return !!this.title && this.title.length > 0;`

Reply:
Also, would be wise to trim the title before the checks